repo/packages/C/ceph/0/config.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
well everything seems pretty explained in detail with copy&paste-ready code to me. What's missing here? https://github.com/fernandosanchezmunoz/examples/blob/c1446aced1ff8096693c9cc6a5e1b7e09d413d2a/1.8/ceph/README.md